apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -414,6 +414,33 @@ const PressReleaseListSectionCodec = t.strict({
   sectionID: t.union([t.string, t.null]),
 });
 
+const MediaResourcesItemCodec = t.strict({
+  title: t.string,
+  thumbnail: t.strict({
+    data: t.strict({
+      attributes: t.strict({
+        url: t.string,
+      }),
+    }),
+  }),
+  resource: t.strict({
+    data: t.strict({
+      attributes: t.strict({
+        url: t.string,
+      }),
+    }),
+  }),
+  icon: StrapiImageRequiredSchema,
+  label: t.string,
+});
+
+const MediaResourcesSectionCodec = t.strict({
+  __component: t.literal('sections.media-resources'),
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  sectionID: t.union([t.string, t.null]),
+  items: t.array(MediaResourcesItemCodec),
+});
```

Comment:
```suggestion
const MediaResourcesSectionCodec = t.strict({
  __component: t.literal('sections.media-resources'),
  theme: t.union([t.literal('light'), t.literal('dark')]),
  sectionID: t.union([t.string, t.null]),
  title: t.union([t.string, t.null]),
  items: t.array(MediaResourcesItemCodec),
});
```
Title was missing. It was missing in Strapi as well, my bad.